src/platform-includes/capture-error/apple.mdx

```diff
@@ -26,3 +26,17 @@ By default, macOS applications do not crash whenever an uncaught exception occur
 1. Open the application's `Info.plist` file
 2. Search for `Principal class` (the entry is expected to be `NSApplication`)
 3. Replace `NSApplication` with `SentryCrashExceptionApplication`
+
+## Customising Error Descriptions
+
+By default, Sentry will display error code as the error description. For custom error types, you may want to provide a custom description that makes it easier to identify the error in the Issues list. You can provide a custom description for `NSError` values by adding a description to the `userInfo` dictionary with the key `NSDebugDescriptionErrorKey`.
+
+This can be particularly useful for Swift enum error types that conform to `Error`, where the error code can be hard to match with an enum case. To customise the description for Swift `Error` types you should conform to the `CustomNSError` protocol and return a user info dictionary:
```

Comment:
```suggestion
This can be particularly useful for Swift enum error types that conform to `Error`, where the error code can be hard to match with an enum case. To customize the description for Swift `Error` types, you should conform to the `CustomNSError` protocol and return a user info dictionary:
```